pytb
"""
test_ignore_listener (test.test_multiprocessing.TestIgnoreEINTR) ... test test_multiprocessing failed -- Traceback (most recent call last):
  File "D:\cygwin\home\db3l\buildarea\2.7.bolen-windows7\build\lib\test\test_multiprocessing.py", line 1524, in test_connection
    conn.send_bytes(really_big_msg)
ValueError: Cannnot send 16777216 bytes over connection
"""
